shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.web.filter;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.springframework.mock.http.server.reactive.MockServerHttpRequest;
+import org.springframework.mock.web.server.MockServerWebExchange;
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.server.WebFilterChain;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * test case for HealthFilter.
+ */
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public final class HealthFilterTest {
+
+    private HealthFilter healthFilter;
+
+    private WebFilterChain webFilterChain;
+
+    @BeforeEach
+    void setUp() {
+        List<String> paths = new ArrayList<>();
+        paths.add("/testFilterMatch");
+        healthFilter = new HealthFilter(paths);
+        webFilterChain = mock(WebFilterChain.class);
+        when(webFilterChain.filter(any())).thenReturn(Mono.empty());
+    }
+
+    @Test
+    void testFilterMatch() {
+        ServerWebExchange webExchange =
+                MockServerWebExchange.from(MockServerHttpRequest
+                        .post("http://localhost:8080/testFilterMatch"));
+        Mono<Void> filter = healthFilter.filter(webExchange, webFilterChain);
+        StepVerifier.create(filter).expectSubscription().verifyComplete();
+    }
+
+    @Test
+    void testFilterNotMatch() {
+        ServerWebExchange webExchange =
+                MockServerWebExchange.from(MockServerHttpRequest
+                        .post("http://localhost:8080/"));
+        Mono<Void> filter = healthFilter.filter(webExchange, webFilterChain);
+        StepVerifier.create(filter).expectSubscription().verifyComplete();
```

Comment:
match and NotMatch, Is it the same verification method?

Reply:
I think the filter method responds to void, so it's all the same   verification method.
i also see the  org.apache.shenyu.web.filter.FallbackFilterTest

Reply:
Looking forward to giving a good suggestion, I can go to learn it

Reply:
hi， if match ， maybe verify this result (or method)

```
   protected Mono<Void> doDenyResponse(final ServerWebExchange exchange) {
        String result = JsonUtils.toJson(new Health.Builder().up().build());
        DataBuffer dataBuffer = exchange.getResponse().bufferFactory().wrap(result.getBytes(StandardCharsets.UTF_8));
        return exchange.getResponse().writeWith(Mono.just(dataBuffer));
    }
```